pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1836,6 +1845,9 @@ protected void handleCloseConsumer(CommandCloseConsumer closeConsumer) {
             consumers.remove(consumerId, consumerFuture);
             commandSender.sendSuccessResponse(requestId);
             log.info("[{}] Closed consumer, consumerId={}", remoteAddress, consumerId);
+            if (getBrokerService().getInterceptor() != null) {
+                getBrokerService().getInterceptor().consumerClosed(this, consumer, null);
```

Comment:
I have a minor question, when will the metadata(the 3rd paramters of `consumerClosed`) not be null, as well as `producerClosed`

Reply:
This should be the metadata for producer or consumer.